documentation/cobigen-intellij-installation.asciidoc

```diff
@@ -0,0 +1,62 @@
+:toc:
+toc::[]
+
+
+= Installation
+
+
+== IntelliJ setup
+
+If you want to use IntelliJ for CobiGen project, you can follow the steps below:
+
+. You can install and create script for IntelliJ as described in
+
+`https://github.com/devonfw/ide/blob/master/documentation/intellij.asciidoc`
+
+. After starting IntelliJ, you can import CobiGen as maven project
+
+. You need now to configure maven settings in IntelliJ as follows
+
+.. Go to File-> Settings -> Build, Execution, Deployment -> Build Tools -> Maven
+
+... Choose Maven home Path from your devonFolder -> devonProjects/software/Maven
+
+... Override user settings file -> devonProjects/conf/.m2/settings.xml
+
+... Override Local repository -> devonProjects/conf/.m2/repository
+
+
+image::images/howtos/intellij/maven-settings[Maven Settings]
+
+.. Now go to File -> Project structure -> Project Settings -> Project add java sdk version from -> devonProjects/software/java
+
+image::images/howtos/intellij/java-sdk.png[Java SDK]
+
+After you have finished your configuration, run maven install and maven clean. CobiGen should build without any errors.
+
+
+=== IntelliJ Testing and Debugging
+
+. Disable ArgLine: If there are any tests which does not work, you might want to disable argline. Go to  File-> Settings -> Build, Execution, Deployment -> Build Tools -> Maven -> Running Tests then uncheck ArgLine
+
+image::images/howtos/intellij/argline.png[Unchecked Argline]
+
+. CLI DEBUGGING
+If you want to debug an application in IntelliJ for example the CLI project, you can go to Run -> Edit Configurations. You can add a new configuration then configure the CLI as shown in the picture
```

Comment:
Better: Simply add a new configuration and configure the CLI as shown in the picture.

Reply:
isn't there simply a link on the web, which could provide a more intense howto? Then it might be also an option to simply link that